### ISSUE TITLE
Video is displayed without detection

### ISSUE THREAD

**Issue by AmirHarvey** [paul-pias/Object-Detection-and-Distance-Measurement#20]
Hello @paul-pias ,
Thank you for sharing your code.

I'm trying to run the code on a video file using cv.VideoCapture() and (ret,frame), the video is displayed as it is without object detection nor distance measurement.
What am i missing here?

**Comment by paul-pias** [comment]
That's a weird problem. Is there any particular error you see on the terminal? The code should work fine. What video format you are currently using? Try changing the format of the input video file.

**Comment by AmirHarvey** [comment]
No, there is no error and the elapsed time and fps are being displayed normally. I was using mp4 and then i tried changing it to avi but still the video is displayed as the original video.

**Comment by rachita2901** [comment]
Hey 
I am facing the same issue. Please help me out if possible.

**Comment by frostybaby** [comment]
Hi @rachita2901! Did you find a solution to the issue?

**Comment by HassanBinHaroon** [comment]
Check out the following relevant and easiest implementation. There happens both. 

https://github.com/HassanBinHaroon/object_detection-PLUS-distance_estimation-v1
